Kconfig

```diff
@@ -184,4 +184,13 @@ menu "Camera configuration"
         config LCD_CAM_CONV_BT709_ENABLED
             bool "BT709"
     endchoice
+
+    config LCD_CAM_CONV_FULL_RANGE_ENABLED
+        bool "Camera converter full range mode"
+        depends on CAMERA_CONVERTER_ENABLED
+        default y
+        help
+            Supports format conversion under both full range mode and limited range mode.
+            If this config item is set, full range mode is used. 
+            Otherwise, The format conversion function will be done using the limited range mode.
```

Comment:
I think it would be helpful to briefly explain here what is the "full range mode" and what is "limited range mode". (I am sure this is mentioned somewhere in the ESP32-S3 TRM, but most users of this library will likely not have read it.

Besides, it will help assess the effect of this PR. Judging by the code below, previously the "limited range mode" was used. After this PR is merged, the "full range mode" will be used since the option is enabled by default.

It would be helpful to explain why (in which case) the user might need to choose one or the other option.

Reply:
OK，relevant instructions have been added to the Kconfig file.  When testing the function for the first time, I only focused on the final data format, not the value of the data. After subsequent tests, I found that the camera sensor we used works in full color range mode, so the full color range mode is used by default in the code.